### PR TITLE
The News Week Clone Homepage Complete 90% Using Bootstrap

### PR DESCRIPTION
**Pull request by Elbie-em**
-Added Nav (use of "vw and not %" in css to exhaust whole viewport due to layout setting)
-Added main section with several articles (ensured all content was corresponding to actual page)
-Added Footer with respective elements

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+}
+
+.c-bg {
+  background-color: #f4f5f7;
+}
+
+.culture-n-travel {
+  display: flex;
+  flex-direction: row;
+}
```
Comment (mosesogwo) [inline styles.css]: Replace with bootstrap class.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+
+.division {
+  width: 100vw;
+}
+
+.subscription-sec,
+.subscription-sec-b {
+  display: flex;
+  flex-direction: row;
```
Comment (mosesogwo) [inline styles.css]: Replace with bootstrap classes

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+  background-color: #f4f5f6;
+  height: 350px;
+  overflow: hidden;
+}
+
+.sub-sec-right {
+  display: flex;
+  flex-direction: column;
```
Comment (mosesogwo) [inline styles.css:194]: Replace with bootstrap classes

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,411 @@
...
+}
+
+.news-sec-right {
+  display: flex;
+  flex-direction: column;
```
Comment (mosesogwo) [inline styles.css:199]: Replace with bootstrap classes

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,411 @@
...
+}
+
+.line {
+  color: #7f7f7f;
+}
+
+.footer-top {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```
Comment (mosesogwo) [inline styles.css]: Replace with bootstrap classes

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,411 @@
...
+    order: 3;
+  }
+
+  .latest-new-sec {
+    width: 100%;
+  }
+
+  .culture-n-travel {
+    display: flex;
+    flex-direction: column;
```
Comment (mosesogwo) [inline styles.css]: Replace with bootstrap classes

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,411 @@
...
+    overflow-y: hidden;
+  }
+
+  .inmag-h {
+    height: auto;
+  }
+
+  .subscription-sec {
+    display: flex;
+    flex-direction: column;
```
Comment (mosesogwo) [inline styles.css]: Replace with bootstrap classes

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,411 @@
...
+    background-color: #f4f5f6;
+    height: 520px;
+    text-align: center;
```
Comment (mosesogwo) [inline styles.css:380]: Replace with bootstrap classe

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,411 @@
...
+  }
+
+  .line {
+    display: none;
+  }
+
+  .footer-top {
+    display: flex;
+    flex-direction: column;
```
Comment (mosesogwo) [inline styles.css]: Replace with bootstrap classes

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,411 @@
...
+  }
+
+  .footer-top-left {
+    display: flex;
+    flex-direction: column;
```
Comment (mosesogwo) [inline styles.css]: Replace with bootstrap classes

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @Elbie-em, you have a nice project:+1:. You need to make the following adjustments:**
1. Remove the vertical scroll in the Featured Stories column.

2. Add some margin above the Latest News and More Stories titles, and on medium and small screens, above the Opinion section also.

3. One of the images in the Editor's Picks section is not being displayed.

4. Reduce left margin on footer for medium and small screen sizes. Compare with the original.

5. On small screen sizes, the More Stories section should come before Culture & Travel section.

6. Also, on small screen sizes, in the Latest News section and the Sections after Featured Slideshow (the three article images under each section) -- Images in these sections appear smaller than it is on the original page. Please compare them.
latest

image

7. The Navbar breaks at a certain point on small screens.
image

8. We could use more bootstrap classes. See comments on your stylesheet.

**Optional:** I think it will appear better to have the header on large screens as not-fixed since it doesn't change on scroll to the minified version being used on small and medium screens.

_**Kindly make these changes and submit another code review request.**_

**Cheers:stars:**

File: styles.css (hunk 10 above)
Comment (Elbie-em) [reply styles.css]: This was created to avoid breaking of the way footer was designed, use of bootstrap breaks the layout drastically

**Comment by Elbie-em** [comment]
1 . Vertical scroll used as that how the page was on Thursday 13th February. Page is bound to change on a daily basis is what I have noticed

2. Margin added as requested

3. Image not showing fixed

4. Margin caused due to use of bootstrap container, changed to fluid and margin reduced by adding margin to container

5. More stories coming before culture and travel fixed

6. Image size increased on smaller resolutions (please note image links are bound to change on the daily if article is removed from homepage, a slight glitch in size may occur if article has been moved, current images are sourced from Friday 14th February"

7. Nav fixed, does not break on smaller resolutions

8. Bootstrap classes implemented where they do not cause breaking of content

Optional: Header not fixed

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to master.
_Goodluck in your next project_ *Happy coding*

**Comment by Elbie-em** [comment]
Thanks alot @pelzolga123

**Pull request merged**